### DISCUSSION TITLE
chia wallet show 0 xch

### DISCUSSION THREAD

**Discussion by gwa1972** [Chia-Network/chia-blockchain#6789] [Support]
despite chiaexplorer.com show me that I won xch, the gui wallet is 0. 
wallet and node are perfect syinc. I am using chia 1.1.7 with laptop ryzen 5 16 gb ram and external 2 ssd 8tb
Please let me know
Thank you for your kind feed back
 
![wallet](https://user-images.githubusercontent.com/85962464/122123057-65411880-cde2-11eb-858e-98ed76865e73.jpg)
![full node](https://user-images.githubusercontent.com/85962464/122123083-6c682680-cde2-11eb-9b00-13790f8e48a5.jpg)

**Comment by loppefaaret** [comment]
could you show me the chia explorer address perhaps ? are you sure the address belongs to your wallet ?

**Comment by BrandtH22** [comment]
@gwa1972 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
